backend/src/test/java/capstoneexpensetracker/backend/TravelerServiceTest.java

```diff
@@ -0,0 +1,27 @@
+package capstoneexpensetracker.backend;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+public class TravelerServiceTest {
+    TravelerRepository travelerRepository = mock(TravelerRepository.class);
+    TravelerService travelerService = new TravelerService(travelerRepository);
+
+    @Test
+    void getTravelerList(){
+        //GIVEN
+        List<Traveler> travelers = new ArrayList<>();
```

Comment:
In diesem Fall ist die veränderbare ArrayList nicht optimal. Der Productioncode könnte die Liste manipulieren (was er nicht soll) und der Test wäre trotzdem grün.

Vorschlag: List.of nutzen (immutable)